tonic/prototype/datasets/ncars.py

```diff
@@ -0,0 +1,110 @@
+from .utils._dataset import Dataset, Sample
+from .utils._utils import check_sha256
+import os
+from expelliarmus import Wizard
```

Comment:
this line is the reason why Tonic as a whole now depends on expelliarmus

Reply:
Uhm, that's no good. Would it be better to include it in the `NCARSReader` class?

Reply:
![image](https://user-images.githubusercontent.com/45739782/201759959-8342f080-9d6c-4e23-8fe7-390ffbb73f02.png)
Like this

Reply:
to be honest I feel like with the work we did, expelliarmus is ready to be a fixed dependency. I'd say include it in the base requirements and delete it from test/docs requirements.

Reply:
Uhm okay, I can feel the pressure now though ahahah

Reply:
In case you prefer to make it a dynamic import I'm also happy with it. Your call!

Reply:
If we break stuff we can always fix it. It's our project we do what we want :P

Reply:
![image](https://user-images.githubusercontent.com/45739782/201761001-2bd9c9ff-8b32-4b23-a9ce-840c74658610.png)

I'll follow the philosophy of my GitHub avatar